superset-frontend/cccs-viz/plugins/plugin-chart-hello-world/src/plugin/controlPanel.ts

```diff
@@ -122,10 +113,50 @@ function validateIP(v: unknown) {
     }
   }
 
-  return ('is expected to be an ip address or cidr');
+  return (' is expected to be an ip address or cidr');
```

Comment:
```suggestion
  return (' is expected to be an IP address in dotted decimal or CIDR notation');
```
Or something like that? Is that leading whitespace necessary for formatting?